### PR TITLE
GitHub Reference feature is in wrong package #1019

### LINKED ISSUE #1019: GitHub Reference feature wrong package
# About

The GitHub reference feature is located in `command.github` but should be in `features.github` instead like any other feature.

### PR DESCRIPTION
**Pull request by Jayanta-karmakar**
fixes #1019 
The GitHub reference feature is located in the `commands.github` but should be in `features.github` instead like any other feature. so, I rebased  `commands.github` to `features.github`, and made some methods public so that they can be accessible.

### CODE REVIEW

**Review by Taz03 — APPROVED** [review]

**Comment by christolis** [comment]
Great job @Jayanta-karmakar! Only thing that's left to do now is to get the Spotless checks to pass. Was the pre-commit hook executed when creating the commit? You should use the command `./gradlew spotlessApply` in order to make them pass!

**Review by christolis — DISMISSED** [review]
LGTM!

**Review by Taz03 — DISMISSED** [review]

**Review by SquidXTV — DISMISSED** [review]
Why is the GitHubCommand.java file still in `commands\github`? The full feature is supposed to be in `features\github`

**Comment by Jayanta-karmakar** [comment]
Okay @SquidXTV, I will Fix it

File: application/src/main/java/org/togetherjava/tjbot/features/github/GitHubReference.java [hunk 1]
```diff
@@ -142,7 +142,7 @@ private void replyBatchEmbeds(List<MessageEmbed> embeds, Message message,
     /**
      * Generates the embed to reply with when someone references an issue.
      */
-    MessageEmbed generateReply(GHIssue issue) throws UncheckedIOException {
+    public MessageEmbed generateReply(GHIssue issue) throws UncheckedIOException {
```
Comment (ankitsmt211) [inline application/src/main/java/org/togetherjava/tjbot/features/github/GitHubReference.java:145]: any reason to make em all  public?

**Review by Taz03 — DISMISSED** [review]

**Review by ankitsmt211 — DISMISSED** [review]
looks good 👍
Thank you 😊

**Review by Taz03 — APPROVED** [review]

**Review by ankitsmt211 — APPROVED** [review]

**Pull request merged**